logging/logging.go

```diff
@@ -0,0 +1,43 @@
+// Package logging provides a logger, as well as configuration.
+package logging
+
+import (
+	"github.com/giantswarm/request-context"
+)
+
+var (
+	logger *requestcontext.Logger
+
+	// The default log level to use.
+	logLevel = "INFO"
+	// Whether to use color, by default.
+	useColor = false
+)
+
+// SetLogLevel sets the log level of the logger to be created.
+func SetLogLevel(level string) {
+	logLevel = level
```

Comment:
@zeisss suggested using the `request-context` library. Our logging discussion on Slack seemed to lean towards using `go-logging` - which this library uses - but using the `request-context` library allows us to also use the context stuff it also has. This is mainly for use later when we move to client-service.

The `logging` package is just a little wrapper around it, mainly so we don't have to set up configs in multiple locations.